packages/babel-plugin/src/__tests__/index.test.ts

```diff
@@ -406,4 +406,75 @@ describe('babel plugin', () => {
 
     expect(actual).toInclude('import { ac, ix, CC, CS } from "@compiled/react/runtime"');
   });
+
+  it.only('should process cssMap even without a react component referencing the map', () => {
```

Comment:
important test to cover transform in runtime. ensured the styles will be generated and grouped with non-atomic classnames. ( before: one rule one classnames, now base -> 1 classname, icon -> 1 classname )